WeatherTodayUITests/Sources/Specs/Application/ApplicationSpec.swift

```diff
@@ -19,12 +24,72 @@ final class ApplicationSpec: QuickSpec {
                 app.terminate()
             }
 
-            context("when opens") {
+            context("when no city name entered") {
 
-                it("empty tests") {
-                    expect(true) == true
+                it("should show default temperature text") {
+                    expect(app.staticTexts[defaultTemperatureText].exists) == true
+                    expect(app.staticTexts[defaultHumidityText].exists) == true
                 }
             }
+
+            context("when invalid city name entered") {
+
+                beforeEach {
+                    let cityNameTextField = app.textFields["Write city name here"]
+                    cityNameTextField.tap()
+                    cityNameTextField.typeText("A")
+                    sleep(1)
+                    app.keyboards.buttons["return"].forceTapElement()
+                }
+
+                it("should show default temperature and humidity text text") {
+                    expect(app.staticTexts[defaultTemperatureText].exists) == true
```

Comment:
FYI: we should put only one `expect` per `it` more detail [here](https://nimblehq.co/compass/development/code-conventions/swift/quick-nimble#convention-6)

Reply:
I think it's ok in UI to assert multiple element in one it because each it will require a rerun of the whole flow, slowing down the flow and increase usage on CICD.

Reply:
@blyscuit I see, so all good now, thanks! 🙏

Reply:
Yes thats why I added multiple expect inside a single it block, but now I disable the CI on bitrise as the credit needs to be saved for release the app to firebase.